azkaban-common/src/test/java/azkaban/executor/MockExecutorLoader.java

```diff
@@ -49,7 +54,8 @@
   @Override
   public void uploadExecutableFlow(ExecutableFlow flow)
       throws ExecutorManagerException {
-    flows.put(flow.getExecutionId(), flow);
+    ExecutableFlow exFlow = ExecutableFlow.createExecutableFlowFromObject(flow.toObject());
```

Comment:
This: https://github.com/juhoautio/azkaban/blob/8272797bf60cf4009e20300e490e7838573a457c/azkaban-common/src/test/java/azkaban/executor/MockExecutorLoader.java#L129

Maybe there could be a class level comment instead telling what the point of this class is (ie. capture status updates so that they can be checked in unit tests).

Should I also add comments in these places where cloning is done why it's needed?